cpp/src/arrow/compute/exec/expression_test.cc

```diff
@@ -258,10 +292,11 @@ TEST(Expression, ToString) {
   EXPECT_EQ(literal(std::make_shared<BinaryScalar>(Buffer::FromString("az"))).ToString(),
             "\"617A\"");
 
-  auto ts = *MakeScalar("1990-10-23 10:23:33")->CastTo(timestamp(TimeUnit::NANO));
-  EXPECT_EQ(literal(ts).ToString(), "1990-10-23 10:23:33.000000000");
+  using namespace arrow_literals;
```

Comment:
This is an unexpected place to find the `using` statement.  Why not pull it outside the test case?

Reply:
I was trying to keep the scope of the literals contained. As it is, not even this was contained enough for lint https://github.com/apache/arrow/actions/runs/3481185552/jobs/5821942926#step:5:903